src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/idam/Credential.java

```diff
@@ -1,7 +1,9 @@
 package uk.gov.hmcts.reform.bulkscan.orchestrator.services.idam;
 
-public class Credential {
+import org.springframework.boot.context.properties.ConfigurationProperties;
 
+@ConfigurationProperties
```

Comment:
does this need to be here?

Reply:
I believe so ... Ill experiment a bit but I think it has to be there for spring to call the setter